docker/Dockerfile-workers

```diff
@@ -6,31 +6,26 @@ RUN dnf -y install \
     --setopt=deltarpm=0 \
     --setopt=install_weak_deps=false \
     --setopt=tsflags=nodocs \
+    httpd \
+    httpd-devel \
```

Comment:
These 2 were only needed in `Dockerfile-api` for the `mod_wsgi` package, workers do not install that one (so you can remove them here)